packages/components/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "6.6.1",
+  "version": "6.6.2",
```

Comment:
current version is 7.0.0, but probably we want to publish this fix for 6.x.x as well